src/test/java/org/ballerinalang/tests/connectors/http/HTTPMethodsTests.java

```diff
@@ -31,6 +35,7 @@
     @Test(description = "Test all methods in one server resource") public void testAllVerbsInOneResource()
             throws IOException {
 
```

Comment:
Don't put a blank line before starting a method definition, fix in all the places.

Reply:
This is fixed with #10